### PR TITLE
feat: 🎸 quit portfolio custody

### PR DESCRIPTION
**Pull request by sansan**
### JIRA Link 

https://polymesh.atlassian.net/browse/DA-78

### Changelog / Description 

- adds endpoint to quit custody over portfolio

### Checklist - 

- [x] New Feature ?
- [x] Updated swagger annotation (if API structure is changed) ?
- [x] Unit Test (if possible) ?
- [x] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/portfolios/portfolios.controller.ts [hunk 1]
```diff
@@ -254,4 +254,42 @@ export class PortfoliosController {
...
+  })
+  @ApiParam({
+    name: 'did',
+    description: 'The DID of the Identity who owns the Portfolio for which Custody is to be quit',
+    type: 'string',
+    example: '0x0600000000000000000000000000000000000000000000000000000000000000',
+  })
+  @ApiParam({
+    name: 'id',
+    description: 'The ID of the portfolio for which to quit Custody. Use 0 for default Portfolio',
```
Comment (polymath-eric) [inline src/portfolios/portfolios.controller.ts:271]:
```suggestion
    description: 'The ID of the portfolio for which to quit Custody. Use 0 for the default Portfolio',
```

(nit)

**Review by polymath-eric — APPROVED** [review]

**Comment by polymath-eric** [comment]
SDK could be upgraded to `v19.0.0-alpha.14` to include the bug fix in quit custody

**Review by VictorVicente — CHANGES_REQUESTED** [review]
@sansan Could you update SDK to latest `beta` in this PR?

**Review by polymath-eric — APPROVED** [review]

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**